pkg/kv/kvserver/kvstorage/destroy.go

```diff
@@ -82,16 +56,9 @@ func clearRangeData(
 		},
 		ReplicatedByRangeID:   opts.clearReplicatedByRangeID,
 		UnreplicatedByRangeID: opts.clearUnreplicatedByRangeID,
```

Comment:
TODO: in the unreplicated keyspace, we don't need to clear "range keys" since they don't exist.

TODO: check if the same applies to the replicated RangeID-local space.